src/lib/components/segment.svelte

```diff
@@ -9,9 +9,20 @@
     }
   }
 
-  export const trackEvent = (event_name: string, props: any) => {
+  const allowsAnalytics = () => {
+    return !!Cookies.get("gp-analytical");
```

Comment:
Let's import the constant from `constants.ts` and use that instead
```suggestion
    return !!Cookies.get(cookies.ANALYTICAL);
```